backend/blueprint/api/controllers/modifyUser.py

```diff
@@ -0,0 +1,45 @@
+from flask import Blueprint, jsonify, request
+from utils.response import Response
+from models.user import User
+from helpers.verifyPassword import verifyPassword
+from werkzeug.security import generate_password_hash
+
+router = Blueprint('modifyUser', __name__)
+
+
+@router.route('/password', methods=['PUT'])
+def changePassword():
+    data = request.get_json()
+    if data and data['username']:
+        user = User.getUser(data['username'])
+        if user:
+            if not verifyPassword(user, data['password']):
+                return jsonify(Response(401).generateMessage('Wrong username and password combination'))
+            user.password = generate_password_hash(data['newPassword'])
+            try:
+                user.save_to_db()
+            except Exception as e:
+                print(e)
+                return jsonify(Response(401).generateMessage('Unable to update password'))
+            return jsonify(Response(200).generateMessage('Password Updated successfully'))
+    else:
+        return jsonify(Response(403).generateMessage('No data received'))
+
+
+@router.route('/name', methods=['PUT'])
+def changeName():
+    data = request.get_json()
+    if data and data['username']:
+        user = User.getUser(data['username'])
+        if user:
+            if not verifyPassword(user, data['password']):
+                return jsonify(Response(401).generateMessage('Wrong username and password combination'))
```

Comment:
Can there be proper indentation just like other files?
Please see this : [https://github.com/fossasia/badgeyay/blob/development/backend/blueprint/api/controllers/generateBadges.py#L9](https://github.com/fossasia/badgeyay/blob/development/backend/blueprint/api/controllers/generateBadges.py#L9)

Reply:
maybe follow the same approach for every Response sent? :sweat_smile: